governance/lead-proposal-process.md

```diff
@@ -0,0 +1,153 @@
+* Existing body of work
+* Justification provided during nomination
+
+In case there are no individuals expressing interest in taking up leadership positions, an evaluation process for the TAG / working group or project archival may be initiated, as [outlined by the TOC](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#retirement).
+
+#### Community timeline
+
+Community nominations will run for at least 1 month, with a maximum up to 2 months.
+
+* For TAG Chair nominations, the TOC will take a vote as [described by the TOC](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections).
+* For TAG TL, TAG Chairs will vote and propose the nomination to the TOC. The TOC takes the final vote.
+* For Working Group and Project Chairs, the TAG Chairs and Leads will take a vote.
+
+Upon close of nominations, a decision should be made within the next month.
+
+#### Submitting a nomination
+
+Nominations are to be submitted to [TAG Co-Chair's email alias](mailto:cncf-tag-env-sus-chairs@lists.cncf.io) with the details: 
+
+* Subject `"NOMINATION $(TAG Chair, TAG TL, WG XYZ Chair): $NAME-OF-NOMINEE"`
+* CC the nominee
+* Justification
+* Nominee Biography
+
+#### Additional note about the TAG Chair proposal process
+
+1. Environmental Sustainability TAG Co-chairs operate as a team. The Co-Chair team seeks to maintain full coverage of the leadership capabilities across the key qualities of industry experience, hands-on cloud-native and sustainability experience, as well as administrative experience needed to run a Environmental Sustainability TAG.
+2. If a clear successor for the exiting Co-Chair exists based on the merit of their contributions ("plant trees, carry water") and relevant experience that will benefit the TAG, based on the TAG's collective experience over time, the exiting Co-chair may nominate a successor to the TOC.
+3. The exiting Co-Chair should propose the desired successor to the other Co-Chairs. If there are any objections, the Co-Chair will have the opportunity to propose another worthy candidate. If Environmental Sustainability TAG Chairs, Tech Leads and TOC Liaisons are unable to put forward a candidate, then a selection process will need to be agreed upon and executed.
+4. If no new Chair candidate is available, the role will remain vacant until the Co-chair's, TOC Liaisons or CNCF TOC propose a replacement.
+
+### General nomination requirements
+
+The TAG ENV recognizes and values the achievements of individuals who have proven themselves in this community.
+If a given nominee does not meet all the requirements, the TAG leadership will still review the nomination and justification to determine if enough intent and commitment is present to move forward with an endorsement.
+
+**Nominees must**:
+
+* Be the author of at least one pull request against the TAG ENV repository
+* Be active in the community within the last 3 months before their nomination
+  * Active - Attending at least 1 TAG ENV Regular Meeting a month
```

Comment:
This may be a bit restrictive, so may be useful to just remove.  There may be cases where they can't make that  time, and maybe we should add a meeting.

Reply:
added